tests/migration_test.go

```diff
@@ -425,6 +493,68 @@ var _ = Describe("Migrations", func() {
 				By("Waiting for VMI to disappear")
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 			})
+			It("should be able successfully cancel a migration", func() {
+
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskFedora))
+				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+				tests.AddUserData(vmi, "cloud-init", fmt.Sprintf(`#!/bin/bash
+					echo "fedora" |passwd fedora --stdin
+					yum install -y stress qemu-guest-agent
+                    systemctl start  qemu-guest-agent`))
+
+				By("Starting the VirtualMachineInstance")
+				vmi = runVMIAndExpectLaunch(vmi, 240)
+
+				getOptions := &metav1.GetOptions{}
+				var updatedVmi *v1.VirtualMachineInstance
+				By("Checking that the VirtualMachineInstance console has expected output")
+				expecter, expecterErr := tests.LoggedInFedoraExpecter(vmi)
+				Expect(expecterErr).To(BeNil())
+				defer expecter.Close()
+
+				// Need to wait for cloud init to finnish and start the agent inside the vmi.
+				Eventually(func() bool {
+					updatedVmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, getOptions)
+					Expect(err).ToNot(HaveOccurred())
+					for _, condition := range updatedVmi.Status.Conditions {
+						if condition.Type == "AgentConnected" && condition.Status == "True" {
+							return true
+						}
+					}
+					return false
+				}, 420*time.Second, 2).Should(BeTrue(), "Should have agent connected condition")
```

Comment:
do we still need to wait 7 minutes here? or can this be reduced?

Reply:
I'm not sure. This is here to make sure that `stress` is installed - I can reduce it, but I don't know how much time really does it take to fetch fedora updates and install it...

Reply:
ah, i see. We're going to have issues with test reliability if we need to depend on installing packages from external repos. Can we deliver the 'stress' binary on a ContainerDisk similar to how the 'virtio-container-disk-image' container disk delivers the virtio binary?

Reply:
Yea, I think we will need to. There are other tests with a similar issue. I'll push a separate PR after this one is merged to improve this, as @rmohr  suggested, somewhere in the comments...

Reply:
yep, makes sense